charts/rsyncd/templates/_helpers.tpl

```diff
@@ -43,3 +43,23 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end -}}
+
+
+{{/*
+Data directory volume definition. Might be defined from parent chart templates or autonomously
+based on the presence of the global value provided by the parent chart.
+Expected argument: dict{
+  "currentRsyncComponent": <string>,
+  "rootContext": { },
+}
+*/}}
+{{- define "rsync.datadir-volumedef" -}}
```

Comment:
I would spell `definition` completely, maybe with a dash to separate it from `volume`.

Reply:
https://github.com/jenkins-infra/helm-charts/pull/830/commits/98ce17924175508f25fe1fc14c94c3d581fb8fd1